index.html

```diff
@@ -0,0 +1,172 @@
+        </nav>
+    </header>
+    <section class="intro-Section">
+        <hgroup>
+            <h1 class="firstHeading">"Hello! Sharing World"</h1>
+            <h1 class="secondHeading">Women in the World Summit 2021</h1>
+    
+        </hgroup>
+        <p>
+            A joufyl celebration believing in the value of opennes and
+            sharing, creating a positive change with people from over 
+            80 countries is taking place in October, in America.
+        </p>
+        <div class="placeTime">
+            <h1>2021.10.11(MON) ~ 15(FRI)</h1>
+            <p>@ National Museum of Korea, Art Center Nabi and more</p>
+        </div>
+    </section>
+    <section class="program">
+        <h1>Main Program</h1>
+        <hr>
+        <div class="programDetail">
+            <ul class="firstDetail">
+                
+                <li><img src="./images/Group.png" alt="Lecture-logo"></li>
+                <li><h1>Lecture</h1></li>
+                <li><p> Listen to the speakers from various countries about the messeges of sharing and opening</p></li>
+
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/group2.svg" alt="Exhibition"></li>
+                <li><h1>Exhibiton</h1></li>
+                <li><p> Passion, leadership, and action are the three themes of this event, which will ultimately lead you to hero status</p></li>
+
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/conversation.svg" alt="Conversation"></li>
+                <li class="headingList"><h1>Forum</h1></li>
+                <li><p> Have the time to share your thoughts
+                    and opinions with experts for
+                    each topic</p></li>
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/workshop.svg" alt="Workshop"></li>
+                <li><h1>Workshop</h1></li>
+                <li><p> It's about shifting the mindset,medium and small scale farming can generate income.</p></li>
+
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/Group.png" alt="Workshop"></li>
+                <li><h1>Equality</h1></li>
+                <li><p> Reducing cultural and gender norms as barriers to equality; changing laws to improve equality</p></li>
+
+            </ul>
+
+        </div>
+        <ul class="buttonList">
+            <li><button type="button">Join Women Summit 2021</button></li>
+        </ul>
+        
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="featuredSpeaker">
+        <h1>Featured Speaker</h1>
+        <hr>
+        <div class="speakerDetail" id="dynamicList">
+        </div>
+        <ul class="buttonList">
+            <li><button class="more-btn" id="more-btn" type="button">More</button></li>
+        </ul>
+        
+    </section>
```

Comment:
- [ ] Good job dynamically rendering the speakers here 👍🏾 
- There's a little tweak to make tho: 
You are missing the image shadow decoration on the top left corner. Kindly add this.

_See images_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/124914629-6ca0af80-dfa5-11eb-96e7-61e860c4ffec.png) | ![image](https://user-images.githubusercontent.com/49621279/124914737-8f32c880-dfa5-11eb-9aa1-b4627a69f0c2.png)

Reply:
- [ ] Good job adding the background shadow to the images, kindly reduce the size of the shadow and the image and make the colour a light grey(`#d3d3d3`).